packages/zambdas/src/ehr/create-update-medication-order/validateRequestParameters.ts

```diff
@@ -41,6 +41,10 @@ export function validateRequestParameters(
 
   validateInteractions(interactions);
 
+  if (orderData.expDate) {
+    orderData.expDate = orderData.expDate.split('T')[0];
```

Comment:
This approach probably shouldn't be used. You're removing timezone information, the date becomes static, and you lose the ability to display it in the user's timezone.

Reply:
The month has the same problem - for example, an expiry date of September 1st could resolve to either September 1st or August 31st depending on the timezone.

Reply:
i think because the time is never submitted (defaults to midnight), this is an issue for most expiry dates. as a user if i see eg `2026-10` as an expiry date, i'll click on the month then click ok, so it'll default to october 1st at midnight and is then susceptible to timezone bugs as usual

Reply:
okay, let's leave it for now and see what happens. there used to be no input validation on this field so anything is an improvement 😅